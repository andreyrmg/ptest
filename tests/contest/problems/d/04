1
663
163 2 39423