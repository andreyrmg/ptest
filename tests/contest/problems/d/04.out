1 237 7
